changelogs/unreleased/2777-fix-error-handling-agent-startup.yml

```diff
@@ -0,0 +1,8 @@
+description: Ensure that an error at agent startup time is properly logged.
+issue-nr: 2777
+change-type: patch
+sections:
+  bugfix: "{{description}}"
+destination-branches:
+  - master
+  - iso4
```

Comment:
Doesn't it go into iso3 too?

Reply:
No. This bug is not present in ISO3 apparently.

Reply:
Oh, indeed :+1:

Reply:
It is rather suspicious however. The bug was introduced in https://github.com/inmanta/inmanta-core/pull/2512. I am wondering why this fix is not present on the ISO3 branch. @wouterdb Do I merge this fix together with https://github.com/inmanta/inmanta-core/pull/2512 on the ISO3 branch?

Reply:
That might be a good idea yes